src/ashley/machina_extensions/forum/admin.py

```diff
@@ -1,6 +1,26 @@
 """
 Import the admin classes from the django-machina's forum application
 """
+from django.contrib import admin
+from machina.apps.forum.admin import ForumAdmin as BaseForumAdmin
+from machina.core.db.models import get_model
 
-# pylint: disable=unused-wildcard-import,wildcard-import
-from machina.apps.forum.admin import *  # noqa isort:skip
+Forum = get_model("forum", "Forum")
+
+admin.site.unregister(Forum)
+
+
+class ForumAdmin(BaseForumAdmin):
+    """The Forum model admin."""
+
+    readonly_fields = ("lti_id",)
```

Comment:
I'm not sure why they need it but they might be interested to search a forum by its lti_id?
```suggestion
    readonly_fields = ("lti_id",)
    search_fields = ['lti_id', other fields?]
```